Src/AutoMoq/StubPropertiesCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// Stubs a mocked object's properties, giving them "property behavior".
+    /// Setting a property's value will cause it to be saved and later returned when the property is accessed.
+    /// </summary>
+    public class StubPropertiesCommand : ISpecimenCommand
+    {
+        /// <summary>
+        /// Stubs a mocked object's properties, giving them "property behavior".
+        /// Setting a property's value will cause it to be saved and later returned when the property is accessed.
+        /// </summary>
+        /// <param name="specimen">The mock to setup.</param>
+        /// <param name="context">The context of the mock.</param>
+        public void Execute(object specimen, ISpecimenContext context)
+        {
+            if (specimen == null) throw new ArgumentNullException("specimen");
```

Comment:
Good question. I never gave it much thought, but I was under the impression `null` was an invalid specimen, seeing as most implementations of `ISpecimenCommand` refuse it - namely, [`AutoPropertiesCommand`](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/Kernel/AutoPropertiesCommand.cs#L148), [`BindingCommand<T, TProperty>`](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/Kernel/BindingCommand.cs#L126) and [`DictionaryFiller`](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/DictionaryFiller.cs#L43).

Personally, I think it makes sense to throw an exception. Commands are executed on the specimen returned by a builder. The builder, as I understand it, should either return an actual value, an instance of `NoSpecimen` or an instance of `OmitSpecimen`. But I can't think of a situation where it would ever return `null`.